manifest.go

```diff
@@ -21,7 +22,8 @@ func LoadManifest() *manifest {
 
 	yfile, err := ioutil.ReadFile("manifest.yaml")
 	if err != nil {
-		log.Fatal().Err(err).Msg("Failed to open file manifest.yaml.")
+		log.Error().Err(err).Msg("Failed to open file manifest.yaml.")
```

Comment:
I think it is better to decrease the level to `Warn`. Also we need to check somewhere if a manifest is manually provided instead and if not, crash there.